<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabSamples">
        <attribute name="title">
         <string>Samples</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="listViewSamples">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineViewSampleFilter">
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnFilterInjection">
             <property name="toolTip">
              <string>Samples with LC/MS Injections</string>
             </property>
             <property name="text">
              <string>I</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/sample_injection_only.svg</normaloff>icons/sample_injection_only.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnFilterFPrint">
             <property name="toolTip">
              <string>Samples with Activity Fingerprints</string>
             </property>
             <property name="text">
              <string>F</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/sample_fingerprint_only.svg</normaloff>icons/sample_fingerprint_only.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAnalyteTables">
        <attribute name="title">
         <string>Analyte Tables</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="listViewAnalyteTables">
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineViewAnalyteTableFilter">
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_mzML"/>
    <addaction name="actionImport_Activity_Fingerprints"/>
    <addaction name="actionImport_Sample_Metadata"/>
    <addaction name="actionImport_Analyte_Table"/>
    <addaction name="separator"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionLoadProject"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionProcess_Monitor"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
   </widget>
   <addaction name="menuImport"/>
   <addaction name="menuView"/>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_mzML">
   <property name="text">
    <string>Import .mzML Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionProcess_Monitor">
   <property name="text">
    <string>Process Monitor</string>
   </property>
  </action>
  <action name="actionImport_Activity_Fingerprints">
   <property name="text">
    <string>Import Activity Fingerprints</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>Load project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImport_Sample_Metadata">
   <property name="text">
    <string>Import Sample Metadata</string>
   </property>
  </action>
  <action name="actionImport_Analyte_Table">
   <property name="text">
    <string>Import Analyte Table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionImport_mzML</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_trigger_import_mzmls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport_Activity_Fingerprints</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_trigger_import_fingerprints()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport_Sample_Metadata</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_trigger_import_metadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport_Analyte_Table</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_trigger_import_analyte_table()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineViewSampleFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_filter_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFilterFPrint</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_filter_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFilterInjection</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>_on_filter_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_on_trigger_import_mzmls()</slot>
  <slot>_on_trigger_import_fingerprints()</slot>
  <slot>_on_trigger_import_metadata()</slot>
  <slot>_on_trigger_import_analyte_table()</slot>
  <slot>_on_filter_changed()</slot>
 </slots>
</ui>
